hack/local-up-cluster.sh

```diff
@@ -908,6 +914,25 @@ function start_kubedns {
     fi
 }
 
+function start_nodelocaldns {
+    if [[ "${ENABLE_NODELOCAL_DNS}" = true ]]; then
+        cp "${KUBE_ROOT}/cluster/addons/dns/nodelocaldns/localdns.yaml.in" nodelocaldns.yaml
+        sed -i -e "s/{{ pillar\['dns_domain'\] }}/${DNS_DOMAIN}/g" nodelocaldns.yaml
+        sed -i -e "s/{{ pillar\['dns_server'\] }}/${DNS_SERVER_IP}/g" nodelocaldns.yaml
+        sed -i -e "s/{{ pillar\['local_dns_ip'\] }}/${LOCAL_DNS_IP}/g" nodelocaldns.yaml
+        if [[ -n "${CLUSTER_IP_RANGE:-}" ]]; then
+            sed -i -e "s@{{ *pillar\['service_cluster_ip_range'\] *}}@${CLUSTER_IP_RANGE}@g" nodelocaldns.yaml
+        else
+            sed -i -e "s@{{ *pillar\['service_cluster_ip_range'\] *}}@0.0.0.0@g" nodelocaldns.yaml
+        fi
+        # TODO update to dns role once we have one.
+        # use kubectl to create nodelocaldns addon
+        ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" --namespace=kube-system create -f nodelocaldns.yaml
+        echo "NodeLocalDNS addon successfully deployed."
+        rm nodelocaldns.yaml
+    fi
```

Comment:
This file is mixed but our style guide says two space indentation:

https://google.github.io/styleguide/shell.xml#Indentation

Reply:
Done